cards.js

```diff
@@ -16,8 +16,45 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
-
+function buildDeck(values, suits){
 // Put some code in here!
+deck =[]
+for(var v = 0; v < values.length; v++){
```

Comment:
I like that you're using counter names that make sense for the arrays you're looping through!